cmd/bnscli/clitests/account_register_domain.test.gold

```diff
@@ -39,7 +38,7 @@
 					}
 				}
 			],
-			"account_renew": 2592000
+			"account_renew": 3840
```

Comment:
Why did this change?

Reply:
The test were not working properly apparently. 12345 h = 44442000 and 1h4m = 3840